### PR TITLE
EOC: References did not come back in time - copy changes

### PR DESCRIPTION
**Pull request by TobyRet**
## Context

From 19 September, when the candidate visits their dashboard, they see a banner if their references were not completed in time.

This banner remains until the user starts a new application (from the interstitial page).

When the candidate clicks ‘Apply again’ from the dashboard banner they see an interstitial page, the copy of which differs depending whether we are between cycles or not, and if Find is closed / open.

### Update copy in the banner

carry_over_banner

#### Start carry over interstitial page (within cycle)

start_carry_over_within_cycle

#### Start carry over page (between cycles)

start_carry_over_between_cycles

## Guidance to review

Give me a yell if you me want to walk through this. 

1. Fired up the app locally and changed cycle to ‘mid cycle’
2. Created a new application with two references
3. Change cycle to ‘Apply 2 deadline has passed’
4. Manually trigger the RejectAwaitingReferencesApplication email. 
5. Go back to application dashboard. A banner should appear. The link that is there (continue your application) should take you to /candidate/application/start-carry-over.

## Link to Trello card

https://trello.com/c/KUyyYMxJ/2132-%F0%9F%9A%B2%F0%9F%94%9A-references-didnt-come-back-in-time-%F0%9F%8F%88

## Things to check

- [x] This code does not rely on migrations in the same Pull Request
- [ ] If this code includes a migration adding or changing columns, it also backfills existing records for consistency
- [ ] API release notes have been updated if necessary
- [ ] New environment variables have been [added to the Azure config](https://github.com/DFE-Digital/apply-for-teacher-training#azure-hosting-devops-pipeline)

### CODE REVIEW

**Comment by TobyRet** [comment]
Woops forgot to update the interstitial page  - doing this now

**Comment by EmmaFrith** [comment]
thanks @TobyRet 
I think the intention with this banner - just comparing to the screenshot here with the Trello card - was to remove the heading 'Do you want to continue applying?' - and the content underneath should be bold
you agree @adamsilver ?

**Comment by EmmaFrith** [comment]
also, looking at it now in context, I think we should state that courses are closed:

# Courses for the 2020 to 2021 academic year are now closed

Your references did not come back in time, but you can [apply again](link to interstitial). 

you agree @adamsilver ?

**Comment by TobyRet** [comment]
> thanks @TobyRet
> I think the intention with this banner - just comparing to the screenshot here with the Trello card - was to remove the heading 'Do you want to continue applying?'
> you agree @adamsilver ?

I've dated the banner

**Comment by TobyRet** [comment]
> also @TobyRet , looking at it now in context, I think we should state that courses are closed:
> 
> # Courses for the 2020 to 2021 academic year are now closed
> Your references did not come back in time, but you can [apply again](link to interstitial).
> 
> you agree @adamsilver ?

Updated banner @EmmaFrith @adamsilver 

carry_over_banner

File: app/views/candidate_interface/submitted_application_form/start_carry_over_between_cycles.html.erb [hunk 1]
```diff
@@ -5,16 +5,36 @@
 <div class="govuk-grid-row">
 
   <div class="govuk-grid-column-two-thirds">
-    <p class="govuk-body">Your last application has been saved, so you do not need to start again.</p>
-    <p class="govuk-body">You can make changes to your application whenever you like.</p>
-    <p class="govuk-body">You will be able to choose courses from <%= EndOfCycleTimetable.find_reopens.to_s(:govuk_date) %> and submit from <%= EndOfCycleTimetable.apply_reopens.to_s(:govuk_date) %>. </p>
-    <p class="govuk-body">You'll have 3 course choices.</p>
 
-    <%= button_to 'Start now', candidate_interface_carry_over_path, class: 'govuk-button govuk-button govuk-!-margin-bottom-5' %>
+    <% if EndOfCycleTimetable.find_down? %>
```
Comment (TobyRet) [inline app/views/candidate_interface/submitted_application_form/start_carry_over_between_cycles.html.erb:9]: @EmmaFrith @adamsilver - in the Trello ticket the requirement is to add this copy if before Oct 6th, but should this actually be the apply 2 deadline (i.e Sep 18th)?

Comment (TobyRet) [reply app/views/candidate_interface/submitted_application_form/start_carry_over_between_cycles.html.erb]: Fixed! 

All date formats now fixed too (they now adopt the YYYY to YYYY format)

**Review by tvararu — APPROVED** [review]
:chefs-kiss:

**Comment by EmmaFrith** [comment]
hey @TobyRet - I'm a bit confused by the page you've called 'Start carry over interstitial page (within cycle)' 

- this shouldn't exist now as far as I can see! 

this scenario is just people people who didn't get their references by the 18 September, so in theory there doesn't need to be an interstitial for 'within cycle'

- the 'Start carry over page (between cycles)' is relevant until a candidate starts a new application, whether it's between cycles or later on next year 

chat tomorrow?

**Pull request merged**